usr/share/rear/skel/NBKDC/etc/scripts/system-setup.d/65-start-nbkdc-client.sh

```diff
@@ -0,0 +1,46 @@
+# start included NBK DC client so file restore can happen:
+
+echo "Sourcing $SHARE_DIR/lib/nbkdc-functions.sh"
+source $SHARE_DIR/lib/nbkdc-functions.sh
+
+# Check if working directories of NBK DC datamover exist
+if [ ! -d $NBKDC_HIBLST_DIR ]; then
+	mkdir $NBKDC_HIBLST_DIR
+fi
+if [ ! -d $NBKDC_HIBTMP_DIR ]; then
+	mkdir $NBKDC_HIBTMP_DIR
+fi
+if [ ! -d $NBKDC_HIBTPD_DIR ]; then
+	mkdir $NBKDC_HIBTPD_DIR
+fi
+if [ ! -d $NBKDC_HIBTAP_DIR ]; then
+	mkdir $NBKDC_HIBTAP_DIR
+fi
+if [ ! -d $NBKDC_HIBMSG_DIR ]; then
+	mkdir $NBKDC_HIBMSG_DIR
+fi
+if [ ! -d $NBKDC_HIBLOG_DIR ]; then
+	mkdir $NBKDC_HIBLOG_DIR
+fi
+
+# Run a checkscript for the datamover to set required soft links
+if [ -e $NBKDC_HIB_DIR/finHib ]; then
+	if [ -x $NBKDC_HIB_DIR/finHib ]; then
+		cd $NBKDC_HIB_DIR && ./finHib
+	else 
+		chmod +x $NBKDC_HIB_DIR/finHib
```

Comment:
chmod +x doesn't hurt so that I would just do that and not check. I am of course assuming that NBKDC has a bug which has the effect that the file ends up without exec permissions. Otherwise: Why do we need this?